### PR TITLE
mitochondrial DNA homoplasmic vs. heteroplasmic assignment #16

### PR DESCRIPTION
**Pull request by abhishek-jain-1999**
Current software is hard-coded to assign homoplasmic vs. heteroplasmic based on: If allelic depth (FORMAT.AD) / read depth (FORMAT.DP) is greater than 99% then allelic state is homoplasmic; else heteroplasmic.

Change the '99%' to a configurable parameter.

### CODE REVIEW

**Comment by abhishek-jain-1999** [comment]
@rhdolin @srgothi92 Please review my PR

File: vcf2fhir/converter.py [hunk 1]
```diff
@@ -70,6 +70,9 @@ class Converter(object):
     (or 0-start, half-open) whereas VCF files and FHIR output are 1-based (or 1-start,
     fully-closed).
 
+    **ratio_ad_dp** (optional)(default value = 0.99): This ratio determine whether to assign Homoplasmic or Heteroplasmic
+     If allelic depth (FORMAT.AD) / read depth (FORMAT.DP) is greater than ratio_ad_dp then allelic state is
+      homoplasmic; else heteroplasmic.
```
Comment (srgothi92) [inline vcf2fhir/converter.py:75]: @rhdolin  does this description looks good to you ? Or there is some bioinformatics way to explain this.

**Review by srgothi92 — APPROVED** [review]
Nice 👍

File: vcf2fhir/converter.py (hunk 1 above)
Comment (rhdolin) [reply vcf2fhir/converter.py:75]: Thanks for this. I'll update manual page with revised text.

Comment (abhishek-jain-1999) [reply vcf2fhir/converter.py:75]: I am sorry , @srgothi92 @rhdolin , I didn't have the full knowledge as to what that parameter actually represented/meant so I just used the issue's explanation to describe it .

**Pull request merged**